build.sc

```diff
@@ -6,8 +6,9 @@ val scalaVersions = "2.11.12" :: "2.12.13" :: "2.13.4" :: "3.0.0-RC1" :: dottyVe
 val scala2Versions = scalaVersions.filter(_.startsWith("2."))
 
 val scalaJSVersions = for {
-  scalaV <- scala2Versions
+  scalaV <- scalaVersions
   scalaJSV <- Seq("0.6.33", "1.4.0")
+  if scalaV.startsWith("2.") || scalaJSV.startsWith("1.")
```

Comment:
just my two cents on DRYness: IMO, making the version tuples explicit here is much easier to understand and also to play around with when editing. We don't really gain much code reuse if we need to subsequently filter many combinations.

Reply:
Yeah, when you have to filter out multiple things it becomes not nice.. I like that makes it harder to forget one combination.
But I think that we'll stop publishing for Scala.js eventually, removing this extra part.

Reply:
no worries, just wanted to mention it because you mentioned DRY. It'd not at all a blocker